shared/js/initApp.js

```diff
@@ -302,9 +309,9 @@ function loadProject(promise) {
   } else if (appOptions.level.projectTemplateLevelName) {
     // this is an embedded project
     dashboard.isEditingProject = true;
-    promise = promise.then(dashboard.loadEmbeddedProject(appOptions.level.projectTemplateLevelName));
+    return dashboard.loadEmbeddedProject(appOptions.level.projectTemplateLevelName);
```

Comment:
Feels a little weird having two different ways of returning a promise in this function. Could probably be cleaned up further.